Dockerfile

```diff
@@ -13,9 +13,8 @@ WORKDIR /node-source
 COPY . .
 
 RUN npm cache verify
-
-RUN npm install && \
-        npm run build
+RUN npm install 
+RUN npm run build
```

Comment:
was this the issue?

Reply:
no it was memory

Reply:
I upgraded the RAM to 2gb and it ran .... lol

Reply:
but it seems like the build passed after this?

Reply:
nah I can trigger a re run without making commits because I set up the pipeline to trigger based on PR hash changes done through new commits or commenting 'rerun' to the PR

Reply:
I pushed this up thinking it was maybe a race issue, but it still failed so I upgraded the ram and ran a rerun and it worked.